client/src/components/modal-forms/ManageGraduationForm.js

```diff
@@ -4,13 +4,31 @@ import React, { useMemo } from 'react';
 import { Box, Grid, Typography } from '@material-ui/core';
 import MuiAlert from '@material-ui/lab/Alert';
 import { Button } from '../../components/generic/Button';
+import { makeStyles } from '@material-ui/core/styles';
 import { AuthContext } from '../../providers';
 import { ParticipantPostHireStatusSchema } from '../../constants/validation';
 import { postHireStatuses } from '../../constants';
 
-export const ManageGraduationForm = ({ initialValues, onClose, onSubmit, cohortEndDate }) => {
+const useStyles = makeStyles(() => ({
+  list: {
+    maxHeight: '100px',
```

Comment:
This is a container with text inside, correct? Generally I'd prefer to see em or rem used here if that's the case. It's a nitpick though.

Reply:
That's an interesting suggestion. I personally tend not to use `em` and `rem` as much but when talking about accessibility they make way more sense. 

As a side note for reference, @mmumby if you consider to follow up on this nitpick consider that:
EM is relative to the parent element’s font size, so if you wish to scale the element’s size based on its parent’s size, use EM.

REM is relative to the root (HTML) font size, so if you wish to scale the element’s size based on the root size, no matter what the parent size is, use REM. If you’ve used EM and are finding sizing issues due to lots of nested elements, REM will probably be the better choice.

My recommendation would be to be consistent with other parts of the application, so whatever we use the most in a specific context ideally would be the way to go.

Reply:
Great discussion - I removed the Alert from this form, and no longer needed this class! I will keep this in mind for sizing in the future though.